Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval/ApprovalTests.Verify_All.ForScenario.Standard_Resources_Resources.generated.approved.cs

```diff
@@ -19118,22 +19118,23 @@ int Entities.Common.EdFi.IChartOfAccount.EducationOrganizationId
             }
         }
 
+        private int _fiscalYear;
+
         /// <summary>
         /// The fiscal year for the account
         /// </summary>
         // IS in a reference, NOT a lookup column 
-        int Entities.Common.EdFi.IChartOfAccount.FiscalYear
+        [DataMember(Name="fiscalYear"), NaturalKeyMember]
+        public int FiscalYear
         {
             get
             {
-                if (ImplicitBalanceSheetDimensionReference != null
```

Comment:
IMO it would be better to return a key unification validation error if the local property is null and a reference is provided (instead of silently taking the value from the reference).
To achieve this, I would have to remove the filter that discards default values in the key unification validation logic.

Reply:
As we agreed with Vinaya, a locally defined property that is key unified shouldn't be optional.